toutv/bos.py

```diff
@@ -58,9 +69,19 @@ def get_proxies(self):
 
     def _do_request(self, url, timeout=None, params=None):
         proxies = self.get_proxies()
+        auth = self.get_auth()
 
         try:
-            r = requests.get(url, params=params, headers=toutv.config.HEADERS,
+            headers = dict(toutv.config.HEADERS)
+
+            if auth and params:
+                url = toutv.config.TOUTV_AUTH_PLAYLIST_URL
```

Comment:
It seems like the V2 URL doesn't work like the V1, to fetch the playlist.

There might be a way to get regular content via the V2 API, I just didn't take the time to play with it.